### PR TITLE
feat(kubernetes): new section

### LINKED ISSUE #301: Kubernetes Page
Kubernetes es una plataforma de código abierto, que fue creada por Google, utilizada para la gestión, el despliegue, la escalabilidad y el monitoreo de contenedores Docker. 

### Características de Kubernetes:
- Distribución inteligente de contenedores.
- Administración de las cargas de trabajos (provee balanceo de carga).
- Fácil gestión de servicios y aplicaciones.
- Alta disponibilidad.
- Gran flexibilidad y permite trabajo modular.
- Monitoreo de contenedores.

### Arquitectura de Kubernetes
Hay dos roles principales, “master” y “node”. El master se encarga de manejar y orquestar los diferentes microservicios entre los 4 nodos disponibles del cluster.

### Esquema de arquitectura de Kubernetes
image

### Conceptos clave en k8s :

**Cluster:** Es un grupo de dispositivos que pueden ser físicos o virtuales y que se encuentran unidos muchas veces por una red y, por lo cual, se comportan como una.

**Nodo Kubernetes:** Es una máquina que se está ejecutando en Kubernetes, en la cual se pueden programar “pods”.

**Pod:** Es un grupo de uno o más containers que comparten el almacenamiento y las configuraciones necesarias para ser ejecutados.

**Replication Controller:** Es quien maneja los fallos antes comentados y recrea, en caso de ser necesario, los pods. Además, también se asegura de que el número de réplicas del pod se esté ejecutando.

**Service:** Es una abstracción que define un conjunto lógico de pods, así como las reglas y la lógica para acceder a ellos.
			

**Infraestructura como Código IaC**
Se creará un repositorio con código para el diseño y despliegue de la infraestructura del proyecto Este representará el estado deseado en los servidores, así como imágenes pre-compiladas de cada servicio que se desea desplegar.

Link a pagina de devops

### PR DESCRIPTION
**Pull request by Edgar097**
### GH Issue

Resolves #301  

### Steps to test
1. Kubernetes Section
1. Check Content

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
LGTM

File: docs/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+id: kubernetes
+title: Kubernetes
+sidebar_label: Kubernetes
+---
+
+Kubernetes is an open source platform, which was created by Google, used for the management, deployment, scalability and monitoring of Docker containers. 
+
+### Kubernetes features
```
Comment (leisterfrancisco) [inline docs/kubernetes.md:9]: Title Case

File: docs/kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+- Intelligent container deployment.
+- Management of workloads (provides load balancing).
+- Easy management of services and applications.
+- High availability.
+- Great flexibility and allows modular work.
+- Container monitoring.
+
+### Kubernetes architecture
```
Comment (leisterfrancisco) [inline docs/kubernetes.md:18]: Title Case

File: docs/kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+There are two main roles, "master" and "node". The master is in charge of managing and orchestrating the different microservices between the 4 available nodes in the cluster.
+
+### Kubernetes Architecture Diagram
+
+![image](https://user-images.githubusercontent.com/5632966/107062957-724c5c00-679f-11eb-88f0-fb4b884785ab.png)
+
+### Key concepts in k8s
```
Comment (leisterfrancisco) [inline docs/kubernetes.md:26]: Title Case

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**